src/components/Profile.jsx

```diff
@@ -17,4 +18,12 @@ class Profile extends React.Component {
 	}
 }
 
+Profile.propTypes = {
+	user: PropTypes.shape({
+		images: PropTypes.array,
```

Comment:
We access `images[0]` without checking for it, so it should be required. It's also an array of what?